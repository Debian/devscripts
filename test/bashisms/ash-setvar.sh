#!/bin/sh

setvar foo bar # BASHISM
[ bar = "$foo" ]
