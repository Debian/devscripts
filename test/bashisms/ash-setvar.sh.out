possible bashism in bashisms/ash-setvar.sh line 3 (setvar 'foo' 'bar' should be eval 'foo="'"$bar"'"'):
setvar foo bar # BASHISM
